src/components/Carousel.js

```diff
@@ -654,14 +655,18 @@ export default class Carousel extends Component {
 
   render() {
     return (
-      <div
-        className={classnames('BrainhubCarousel', this.getProp('className'))}
-        ref={el => this.node = el}
-      >
-        {this.renderArrowLeft()}
-        {this.renderCarouselItems()}
-        {this.renderArrowRight()}
-        {this.renderDots()}
+      <div>
+        <div
+          className={classnames('BrainhubCarousel', this.getProp('className'))}
+          ref={el => this.node = el}
+        >
+          {this.renderArrowLeft()}
+          {this.renderCarouselItems()}
+          {this.renderArrowRight()}
+        </div>
+        <div className={classnames('BrainhubCarousel__dots', this.getProp('className'))}>
```

Comment:
1. The whole div should not render if `!this.getProp('dots')` as the className provided may add some margin/padding or other unexpected changes.
2. I would consider having a separate props for dots className (like `dotsClassName`). It's common to just throw some styles at the carousel without thinking that the class provided in that prop is actually applied to two things (which is quite unintuitive I think)